components/release/base/cronjobs/remove-expired-releases.yaml

```diff
@@ -30,7 +30,7 @@ spec:
                   NOW=$(date +%s)
                   kubectl get release --all-namespaces \
                   --sort-by=.status.expirationTime \
-                  --template '{{range .items}}{{.metadata.name}}{{"\t"}}{{.metadata.namespace}}{{"\t"}}{{.status.expirationTime}}{{"\n"}}{{end}}' > /tmp/kubectl-out
+                  --template '{{range .items}}{{.metadata.name}}{{"\t"}}{{.metadata.namespace}}{{"\t"}}{{.status.expirationTime}}{{"\n"}}{{end}}' > /var/tmp/kubectl-out
```

Comment:
You will also need to change the other place that reads the file. But also, I'd suggest using `mktemp`.
```
OUTPUT=$(mktemp)
.. > $OUTPUT
...
awk ... $OUTPUT
```

Reply:
So you're saying plain `mktemp` won't work here, right? Fine in that case.

Reply:
exactly. thanks!